addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -309,25 +309,27 @@ function makeActionManager(env) {
         const options = { clearBreadcrumbs: true };
         let actionRequest = null;
         if (state.action) {
+            const context = { params: state };
```

Comment:
I think this params key in the context is a mistake. Do you need it? This is something I want to investigate for a while. So if you don't need it, please remove it (I think active_id and active_ids are enough here)

Reply:
No I don't need it either. I just wanted to stay as close as the original code as possible. I'll try to remove it then.